### ISSUE TITLE
Optimization issues

### ISSUE THREAD

**Issue by mio12333** [34736384/genshin-fps-unlock#99]
After replacing my 5700 with a 6950xt, I was able to get a stable 144 in closed scenes like Abyss.

But the fps increase in the big world is very small and the hardware usage is very low, does that mean the game is not optimized well?

If I upgrade my cpu to 5800x3d will it have an effect?

(currently a 3800x)

**Comment by KiritoVeres** [comment]
I imagine getting a better processor would help,

Overall the game is unoptimized'ish, I have a ryzen 7 3700x and 2080s and there's a spot or three in sumeru which makes me drop below 60

**Comment by RedSSSSS** [comment]
> I imagine getting a better processor would help,
> 
> Overall the game is unoptimized'ish, I have a ryzen 7 3700x and 2080s and there's a spot or three in sumeru which makes me drop below 60

can you try setting ''environment detail'' to lowest ? I had framerate issues and none of the settings helped except this one. I honestly think environment on this game works so unoptimized with higher framerates.

**Comment by patricklai3** [comment]
Yes, I have 5800x3d and 6800 xt, locked it at 180 fps to match my monitor but it can drop down to 90 fps at the highest environmental detail. But high environmental details allow me to lock 180 at all times and it can run up to 300+ fps in open areas.

**Comment by Iammethe24** [comment]
Unfortunately Genshin is very CPU bound and it's pretty unoptimized, especially in Sumeru City and the blue mushroom forest thing. My CPU usage is pegged at 100% and I get framedrops to the 50s. The GPU is not really utilized. This can be improved by reducing environmental detail but I can't stand the very obvious pop-in while gliding from high areas. Actually, on highest settings you still get pop in but the pop in you get is the effect of the game switching to a high-detail model, not low-detail models appearing a distance away.
